dts/msm8916/msm8239-samsung-r08.dts

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <263 0>;
+	qcom,board-id = <0x10708FF1 8>;
```

Comment:
I'd still like to know what happens if you don't add this DTS at all. Not sure if you ever answered that question in the chat.

Reply:
Without `msm8239-samsung-r08.dts`, bootloader will ignore `msm8239-samsung-r09.dts`.

Reply:
But iirc the stock image does not have an -r08 either? Anyway, it's fine I guess.

Reply:
Same happens to Grand Prime SM-G530FZ in fd4418246dee2b272b992952d94895511d1d06f7.
There is no `msm8916-sec-fortuna-r02.dts` either but the bootloader chooses to ignore `msm8916-samsung-r03.dts`